sources/platform/actors/development/source_code.md

```diff
@@ -22,6 +22,8 @@ See [Custom Dockerfile](./source_code.md) and [base Docker images](./base_docker
 
 ## [](#git-repository)Git repository
 
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/NEzT_p_RE1Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
maybe better to use full width? 

```suggestion
<iframe width="100%" height="315" src="https://www.youtube-nocookie.com/embed/NEzT_p_RE1Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Reply:
I tried. It crops the video weirdly, unless you also unlock the height. But when you unlock the height, it becomes super huge.